beaconChain/tests/eth1/deploy.ts

```diff
@@ -0,0 +1,72 @@
+import BN from "bn.js";
+import { assert } from "chai";
+import { ethers } from "ethers";
+import ganache from "ganache-core";
+import sinon from "sinon";
+
+import { Eth1Notifier } from "../../src/eth1";
+import defaults from "../../src/eth1/defaults";
+import promisify from "promisify-es6";
+
+// TODO integrate this into longer running tests
+describe("", () => {
```

Comment:
can we get a description :)

Reply:
oh yea! forgot about that. I'll add one.

This is the test file that doesn't run.
Originally, I was planning on actually testing triggering Deposits and an Eth2Genesis from a testnet. But in order to test the Eth2Genesis using the standard contract, 65536 deposits have to be sent. It looked like it was going to take over an hour to process these txs using this `ganache` provider. Even without the Eth2Genesis test, just deploying the deposit contract and testing the Deposit log took "too long" for our current test suite, 30+ seconds added. We'd be pulling out our hair every time we ran it.

I think ideally, this file can get resurrected as part of some longer running test suite. We can deploy two contracts, the standard deposit contract, and a modified deposit contract that will output an Eth2Genesis log w/o requiring all of the txs.